lib/cometbft-types/src/crypto/public_key.rs

```diff
@@ -1,13 +1,18 @@
 use serde::{Deserialize, Serialize};
+use unionlabs::{bytes::Bytes, hash::hash_v2::Base64};
 
 // TODO: These are fixed sizes, not arbitrary bytes
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(tag = "@type", content = "@value", rename_all = "snake_case")]
+#[serde(tag = "type", content = "value", rename_all = "snake_case")]
 pub enum PublicKey {
-    Ed25519(#[serde(with = "::serde_utils::hex_string")] Vec<u8>),
-    Secp256k1(#[serde(with = "::serde_utils::hex_string")] Vec<u8>),
-    Bn254(#[serde(with = "::serde_utils::hex_string")] Vec<u8>),
-    Bls12_381(#[serde(with = "::serde_utils::hex_string")] Vec<u8>),
+    #[serde(rename = "tendermint/PubKeyEd25519")]
+    Ed25519(Bytes<Base64>),
+    #[serde(rename = "tendermint/PubKeySecp256k1")]
+    Secp256k1(Bytes<Base64>),
+    #[serde(rename = "cometbft/PubKeyBls12_381")]
```

Comment:
why this doesn't have tm alias but bn254 does?

Reply:
compatability with union-testnet-8: https://rpc.testnet-8.union.build/validators

<img width="494" alt="image" src="https://github.com/user-attachments/assets/43e565ca-0f11-4a93-aa10-41ca03f36368">